docs/PowerAuth-Push-Server-1.8.0.md

```diff
@@ -0,0 +1,16 @@
+# Migration from 1.7.x to 1.8.x
+
+This guide contains instructions for migration from PowerAuth Push Server version `1.7.x` to version `1.8.x`.
+
+
+## Database Changes
+
+For convenience, you can use liquibase for your database migration.
+
+**Important: Upgrading to version 1.8.x includes column renaming, resulting in incompatible changes. Therefore, ensure all Push Server nodes are upgraded simultaneously during a scheduled service window.**
```

Comment:
I suggest stating that this is for the Inbox feature. For proper impact analysis.

Reply:
Good idea.